cmd/broker/ingress/main.go

```diff
@@ -76,15 +85,26 @@ func main() {
 		logger.Fatal("Unable to add eventingv1alpha1 scheme", zap.Error(err))
 	}
 
-	brokerName := getRequiredEnv("BROKER")
+	brokerName := utils.GetRequiredEnvOrFatal("BROKER")
```

Comment:
Now that we are changing this, I think it would be nice to move to kelseyhightower for the env variables. 
You can see an example in the cronjob receive_adapter main.go. I think it will be cleaner.

Reply:
I agree, as we add more and more env vars, [Kelsey's envconfig](https://github.com/kelseyhightower/envconfig) seems better and better. I'll make an issue for it, but would prefer not to make the change in this PR.

Reply:
#1291